src/accounts/AccountHeader.tsx

```diff
@@ -10,6 +11,7 @@ type Props = {
 const AccountHeader: FC<Props> = ({testID = 'member-page--header'}) => (
   <Page.Header fullWidth={true} testID={testID}>
     <Page.Title title="Account" />
+    <LimitChecker />
```

Comment:
I added this component back in to the Account Header and billing page because it's actually doing more than acting as a wrapper for `RateLimitAlert`. 

On mount, the component also runs the `getReadWriteCardinalityLimits` thunk, which retrieves and updates the status of the user's read, write, and cardinality limits; and dispatches a notification to the user if they've exceeded their read limit. (Makes sense that this would happen when checking the status of the account, or the billing page.)

LimitChecker's props presumed child components, so I modified it slightly to make the rendering of child components optional and turn it into a singleton tag. This way, we can keep it on the page, in the same structure, without needing it to contain another component.